src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -190,9 +141,16 @@ class ValidationStep extends React.Component {
                     </View>
                 )}
                 {!maxAttemptsReached && state === BankAccount.STATE.PENDING && (
-                    <ReimbursementAccountForm
-                        reimbursementAccount={this.props.reimbursementAccount}
+                    <Form
+                        formID={ONYXKEYS.FORMS.VALIDATION_STEP_FORM}
+                        submitButtonText={
+                            currentStep === CONST.BANK_ACCOUNT.STEP.VALIDATION
+                                ? this.props.translate('validationStep.buttonText')
+                                : this.props.translate('common.saveAndContinue')
+                        }
                         onSubmit={this.submit}
+                        validate={this.validate}
+                        style={[styles.mh5, styles.mb5]}
```

Comment:
The submit button should be at the bottom of the page:
```suggestion
                        style={[styles.mh5, styles.flexGrow1]}
```